src/tir/schedule/primitive/loop_transformation.cc

```diff
@@ -482,7 +482,7 @@ StmtSRef Fuse(ScheduleState self, const Array<StmtSRef>& loop_srefs) {
   Array<PrimExpr> substitute_value;
   substitute_value.resize(loops.size());
   PrimExpr tot = fused_var;
-  for (int i = static_cast<int>(loops.size()) - 1; i >= 0; i--) {
+  for (int i = loops.size() - 1; i >= 0; i--) {
```

Comment:
We shouldn't make this change, as when `loops` is empty, `loops.size() - 1` is an unexpected value.

Reply:
size_t i

Reply:
No we should keep with `static_cast` instead. As a principle, `size_t` is all evil and should be avoided unless we have good reasons. In our particular case, if `loops.size()` is 0, underflow of `size_t` it may cause tremendous trouble. Even if by reading around we might have the assumption that `loops.size() > 0`, it is still not as convenient as just casting to signed integers.

Reply:
You are right，didn't notice i starts from loops.size()